Traceback (most recent call last):
  File "/Users/emarshall/miniconda3/envs/mynewbook/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/Users/emarshall/miniconda3/envs/mynewbook/lib/python3.10/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/Users/emarshall/miniconda3/envs/mynewbook/lib/python3.10/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/Users/emarshall/miniconda3/envs/mynewbook/lib/python3.10/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/Users/emarshall/miniconda3/envs/mynewbook/lib/python3.10/asyncio/base_events.py", line 646, in run_until_complete
    return future.result()
  File "/Users/emarshall/miniconda3/envs/mynewbook/lib/python3.10/site-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/Users/emarshall/miniconda3/envs/mynewbook/lib/python3.10/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/Users/emarshall/miniconda3/envs/mynewbook/lib/python3.10/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
attributes = {'id': 11909, 'name':'sample_glacier','valid':True}
centerline = Centerline(sample_glacier_series_prj['geometry'], **attributes)
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mNameError[0m                                 Traceback (most recent call last)
Input [0;32mIn [8][0m, in [0;36m<cell line: 2>[0;34m()[0m
[1;32m      1[0m attributes [38;5;241m=[39m {[38;5;124m'[39m[38;5;124mid[39m[38;5;124m'[39m: [38;5;241m11909[39m, [38;5;124m'[39m[38;5;124mname[39m[38;5;124m'[39m:[38;5;124m'[39m[38;5;124msample_glacier[39m[38;5;124m'[39m,[38;5;124m'[39m[38;5;124mvalid[39m[38;5;124m'[39m:[38;5;28;01mTrue[39;00m}
[0;32m----> 2[0m centerline [38;5;241m=[39m [43mCenterline[49m(sample_glacier_series_prj[[38;5;124m'[39m[38;5;124mgeometry[39m[38;5;124m'[39m], [38;5;241m*[39m[38;5;241m*[39mattributes)

[0;31mNameError[0m: name 'Centerline' is not defined
NameError: name 'Centerline' is not defined

